src/components/ArmyBuildPane.jsx

```diff
@@ -4,15 +4,19 @@ import styled from 'styled-components'
 
 import ArmyList from'./ArmyList';
 import SelectionPane from './SelectionPane'
+import UsageBar from './UsageBar'
 
 const Container = styled.div`
-    display: flex;
+    display: grid;
+    grid-template-columns: 33% 66%
+    grid-template-rows: 10% 90%;
     height: 70vh;
     margin: 30px
 `;
 
 const ArmyBuildPane = () => (
     <Container>
+        <UsageBar/>
```

Comment:
Super pedantic but like, you don't have a space between UsageBar and /> and the other two have it and it's driving me nuts